src/pykiso/lib/connectors/cc_pcan_can/cc_pcan_can.py

```diff
@@ -491,5 +491,5 @@ def _remove_offset(
 
     def shutdown(self) -> None:
         """Destructor method."""
-        if self.logging_activated:
+        if self.logging_activated and self.is_fd:
```

Comment:
Is it really a viable solution to just disable trc merging for non-FD CAN?

Reply:
Where is the bug from? PeakCAN Lib?

Reply:
It is not really a bug, just not supported in their read/write classes. It just mean that we do not handle the merging for some of the older version of trc format.  I updated the pull request so that only those formats are filtered.

Reply:
I checked with them, it is working as intended